templates/experiments/single_experiment_home.html

```diff
@@ -61,25 +61,6 @@ <h1 class="pg-title">{{ experiment.name }}</h1>
               <i class="fa-solid fa-pencil"></i>
             </a>
           </div>
-          <div class="tooltip" data-tip="Delete">
-            <button
-              id="action-delete-{{ experiment.id }}"
-              class="btn btn-primary join-item rounded-r-full"
```

Comment:
I think yo should move the `rounded-r-full` class to the previous button (the one that's now on the end)